lib/matplotlib/axes/_axes.py

```diff
@@ -246,6 +272,22 @@ def set_ylabel(self, ylabel, fontdict=None, labelpad=None, **kwargs):
         """
         if labelpad is not None:
             self.yaxis.labelpad = labelpad
+        _protected_kw = ['y', 'horizontalalignment', 'ha']
+        if any([k in kwargs for k in _protected_kw]):
+            if loc is not None:
+                raise TypeError('Specifying *loc* is disallowed when any of '
+                               'its corresponding low level kwargs {} '
+                               'are supplied as well.'.format(_protected_kw))
```

Comment:
alignments are messed up, flake8 may not like that (likewise below)

Reply:
Yeah, I'm fixing it, figured letting the CI run was the fastest way. Gimme a sec